pandas/tests/groupby/transform/test_numba.py

```diff
@@ -202,6 +202,7 @@ def func_kwargs(values, index):
 
 
 @td.skip_if_no("numba")
+@pytest.mark.filterwarnings("ignore")
```

Comment:
can we just ignore numba-produced warnings?

Reply:
Yeah the `NumbaPerformanceWarning` are safe to ignore. Unfortunately I don't think specifying a `"ignore::NumbaPerformanceWarning"` can be put in the `pyproject.toml` because not all builds have `numba` installed.